Attendee.js

```diff
@@ -0,0 +1,39 @@
+// The Attendee class can be utilized to keep certain 
+// information of the attendee, a class for the user 
+// needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
```

Comment:
Style. Wrong format for class comments. See https://google.github.io/styleguide/jsguide.html#jsdoc-property-comments